cucumber/api/features/support/rest_helpers.rb

```diff
@@ -144,6 +145,13 @@ def authn_local_request command
     end
   end
 
+  def authn_request(url:, api_key:, encoding:, can:)
+    headers["Accept-Encoding"] = encoding ? "base64" : ""
```

Comment:
shouldn't this be nil in case the encoding is not `base64`? something like:
```
if encoding
  headers["Accept-Encoding"] = "base64"
end
```

Reply:
Tried to avoid codeclimate message by put a value for `else`. I will revert this change.